sdk/webpubsub/Azure.Messaging.WebPubSub/api/Azure.Messaging.WebPubSub.netstandard2.0.cs

```diff
@@ -63,6 +65,8 @@ public partial class WebPubSubServiceClient
         public virtual System.Threading.Tasks.Task<Azure.Response> RemoveConnectionFromAllGroupsAsync(string connectionId, Azure.RequestContext context = null) { throw null; }
         public virtual Azure.Response RemoveConnectionFromGroup(string group, string connectionId, Azure.RequestContext context = null) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response> RemoveConnectionFromGroupAsync(string group, string connectionId, Azure.RequestContext context = null) { throw null; }
+        public virtual Azure.Response RemoveConnectionsFromGroups(string filter, IEnumerable<string> groups, Azure.RequestContext context = null) { throw null; }
```

Comment:
filter is an optional parameter, when filter is null, it means remove all the connections from the specified groups

Reply:
fixed